src/node/desktop/src/ui/utils.ts

```diff
@@ -1,3 +1,5 @@
+import path from "path";
```

Comment:
Is this import still needed? The comment below suggests `path` cannot be used here.

Reply:
Removing! Thanks for noticing. Forgot to remove from there